man/adoc/bpftrace.adoc

```diff
@@ -2499,6 +2503,9 @@ kprobe:dummy {
   delete(@associative, (1,2)); // ok
   delete(@associative); // error
   delete(@associative, 1); // error
+
+  // deprecated but ok
+  delete(@associative[1, 2], @single["hello"]);
```

Comment:
```suggestion
  delete(@associative[1, 2]);
  delete(@single["hello"]);
```
Could we show the single map variant instead? It's much older than the multi-one and is what most people will be familiar with.

We'll also be maintaining backwards compatibility with the single variant for a while, but the multi-one I think will have to go soon as it conflicts with the new syntax.

Reply:
Sure. Will update.